astropy/io/fits/column.py

```diff
@@ -764,6 +792,47 @@ def name(col, name):
                 'characters, though it may be fewer if the string '
                 'contains special characters like quotes.')
 
+    @ColumnAttribute('TCTYP')
+    def coord_type(col, coord_type):
+        if coord_type is None:
+            return
+
+        if (not isinstance(coord_type, string_types)
+                or len(coord_type) > 8):
+            raise AssertionError(
+                'Coordinate/axis type must be a string of atmost 8 '
+                'characters.')
+
+    @ColumnAttribute('TCRPX')
+    def coord_ref_point(col, coord_ref_point):
+        if (not coord_ref_point is None
```

Comment:
Use `is not None` instead, it's more clear and I think it is the PEP8 way. (And the same for the other properties below)